src/lib/Header.svelte

```diff
@@ -4,11 +4,18 @@
 	import { flyX } from '$lib/transitions';
 	import MetaNav from './MetaNav.svelte';
 	import HeaderBar from './HeaderBar.svelte';
+	import { page } from '$app/stores';
 	let open = false;
+
+	$: console.log($page.url);
```

Comment:
this line will log the page url. i think you can remove this line :) 

maybe instead you can add a comment instead that this color is for being able to provide a visual clue to differ whether we are on a staging/test or production environment?